### PR TITLE
Stepper

### PR DESCRIPTION
**Pull request by roblevintennis**
**Description:**

- [Asana Task](https://app.asana.com/0/1147552137426708/1171609519486697/f)
- Creating a navigation component -- it's just a dumb vertical stepper; state is pushed in from outside.

_Note that at this point it isn't interactive per the state of the funnel at the moment. When we build in user interaction we'll likely use `<button>` elements and change some of the a11y -- my research indicated we need to not use `<nav>` until it's interactive fwiw._

This is going to be used for something like the following:

image

- [x] Have exported the component from the main [index.js](https://github.com/getethos/ethos-design-system/blob/master/src/components/index.js)?
- [x] Have you exported it from the [index.d.ts](https://github.com/getethos/ethos-design-system/blob/master/src/components/index.d.ts) so Typescript users can consume it? Added a test and ran `yarn test:types`?
- [x] Bumped the yarn version?

**Referencing PR or Branch (e.g. in CMS or monorepo):**

- Corresponding PR: https://github.com/getethos/ethos/pull/3231
Seems to be working from the app shell:
image

**Screenshots:**

These are tests within the EDS:

image

image

image

I also seem to be able to arbitrarily add steps and it grows dynamically as expected:

image

### CODE REVIEW

File: src/components/index.tests.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import * as React from 'react'
-import { Accordion } from './index'
+import { Accordion, Stepper } from './index'
```
Comment (stephaniel96) [inline src/components/index.tests.tsx:2]: Is there a reason you've opted to import `Stepper` and `Accordion` in the same line?

Comment (roblevintennis) [reply src/components/index.tests.tsx:2]: Lol, no 🤦 I'll fix that thanks Steph

Comment (roblevintennis) [reply src/components/index.tests.tsx:2]: I'm actually suspicious that vscode auto did that lol

Comment (roblevintennis) [reply src/components/index.tests.tsx]: fixed in v1.1.187

**Pull request merged**